frontend/app/src/components/landingpages/blog/Governance.svelte

```diff
@@ -0,0 +1,122 @@
+        Other than "Motion" proposals all other types result in some automatic action by the SNS such as...
+    </p>        
+    <ul>
+        <li>Transfer ICP or CHAT out of the treasury</li>
+        <li>Upgrade an OpenChat canister</li>
+        <li>Change SNS meta data such as the "proposal rejection cost"</li>
+        <li>Call a custom SNS function for example to add a "platform moderator"</li>
+    </ul>
+    <p>
+        Join <Link path="/nsbx4-4iaaa-aaaar-afusa-cai">this group</Link> to view, discuss, and vote on OpenChat SNS proposals.
+    </p>
+    <p>
+        Anyone who owns a neuron can use it to make proposals although this currently requires knowledge of the command line tools <ExternalLink href="https://github.com/dfinity/sdk">dfx</ExternalLink> and <ExternalLink href="https://github.com/dfinity/quill/blob/master/docs/cli-reference/sns/quill-sns-make-proposal.md">quill</ExternalLink>. In time this process should become easier. If a proposal is rejected then 10 CHAT tokens is subtracted from the proposing neuron and burned to discourage spam.
+    </p>
+</section>
+<section>
+    <h2>Voting</h2>
+    <p>
+        You can vote to "adopt" or "reject" any proposals with any eligible neurons you own. This allows you take part in the governance of OpenChat and help steer it in a direction of your choosing while also earning voting rewards - see bottom section. To be eligible to vote on a given proposal a neuron must exist and have a dissolve delay of at least 1 month at the time the proposal was made.
+    </p>
+    <p>
+        For any given proposal, the weight of your vote is calculated as the ratio of the <em>voting power</em> of your neurons compared to all other eligible neurons. Voting power for a neuron is calculated as (stake x dissolve delay bonus x age bonus). The dissolve delay bonus starts at 1 for a neuron with 0 dissolve delay and rises linerarly to 2 for neurons with a dissolve delay of 1 year. The age bonus starts at 1 for neurons with 0 age, climbing to 1.25 for neurons with an age of 6 months. To <em>age</em> a neuron must be kept in the undissolving state but as soon it starts dissolving the age resets to 0. For example if you have a neuron with a stake of 1000 CHAT tokens, a dissolve delay of 1 year, and an age of 6 months its voting power will be 2500.
+    </p>
+    <p>
+        For a proposal to be adopted the sum of the voting power of the “adopt” votes must exceed the sum of the voting power of the “reject” votes after 4 days or after a period of quiet. Also the “adopt” votes must have at least 3% of the total voting power. The vote will end immediately if either the “adopt” or “reject” votes exceed 50% of available voting power. It is possible to participate in a vote for a proposal if still within the 4 day period even if it has already been resolved. This is so the neuron can still be a beneficiary of voting rewards.        
+    </p>
+</section>
+<section>
+    <h2>Vote using OpenChat</h2>
```

Comment:
I think maybe we could do with a screen shot / animated sequence here and also for the following section. I can try to come up with something if you like?

Reply:
Sounds good!